CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [3.1.0](https://github.com/auth0/samlp-logout/compare/v3.0.0...v3.1.0) (2021-08-30)
```

Comment:
Any specific reason for a major here ?

Reply:
This is a behavior change, not a "fix"/patch, but I'm OK either way. Let me know if you want to change it to a patch version.

Reply:
That makes sense.